tapdb/assets_store.go

```diff
@@ -337,6 +337,9 @@ type ActiveAssetsStore interface {
 	// FetchAssetMetaForAsset fetches the asset meta for a given asset.
 	FetchAssetMetaForAsset(ctx context.Context,
 		assetID []byte) (sqlc.FetchAssetMetaForAssetRow, error)
+
+	// AssetsDBSize returns the total size of the taproot assets database.
+	AssetsDBSize(ctx context.Context) (int64, error)
```

Comment:
Perhaps we should move this to another interface? So like `Meta` which could also be extended to return information about the current database migration.